From 3d8ab91da6a76373fd54ec107a0114b467f107fb Mon Sep 17 00:00:00 2001
From: Fancy Fang <chen.fang@freescale.com>
Date: Wed, 8 Apr 2015 17:16:43 +0800
Subject: [PATCH 0352/1221] MLK-10593-1 ARM: dts: imx7d-12x12-arm2: correct
 lcdif pad configuration

Correct the lcdif pad configuration according to 7d
reference manual to make the lcd panel display with
no errors.

Signed-off-by: Fancy Fang <chen.fang@freescale.com>
[zou:Original patch taken from
git.freescale.com/imx/fsl-arm-yocto-bsp.git-b imx-3.14.52-1.1.0_ga]
Signed-off-by: zou cao <cao.zou@windriver.com>
---
 arch/arm/boot/dts/imx7d-12x12-arm2.dts |   56 ++++++++++++++++----------------
 1 files changed, 28 insertions(+), 28 deletions(-)

diff --git a/arch/arm/boot/dts/imx7d-12x12-arm2.dts b/arch/arm/boot/dts/imx7d-12x12-arm2.dts
index 93e0732..801db1a 100644
--- a/arch/arm/boot/dts/imx7d-12x12-arm2.dts
+++ b/arch/arm/boot/dts/imx7d-12x12-arm2.dts
@@ -247,39 +247,39 @@
 	lcdif {
 		pinctrl_lcdif_dat: lcdifdatgrp {
 			fsl,pins = <
-				MX7D_PAD_LCD_DATA00__LCD_DATA0	0x4001b0b0
-				MX7D_PAD_LCD_DATA01__LCD_DATA1	0x4001b0b0
-				MX7D_PAD_LCD_DATA02__LCD_DATA2	0x4001b0b0
-				MX7D_PAD_LCD_DATA03__LCD_DATA3	0x4001b0b0
-				MX7D_PAD_LCD_DATA04__LCD_DATA4	0x4001b0b0
-				MX7D_PAD_LCD_DATA05__LCD_DATA5	0x4001b0b0
-				MX7D_PAD_LCD_DATA06__LCD_DATA6	0x4001b0b0
-				MX7D_PAD_LCD_DATA07__LCD_DATA7	0x4001b0b0
-				MX7D_PAD_LCD_DATA08__LCD_DATA8	0x4001b0b0
-				MX7D_PAD_LCD_DATA09__LCD_DATA9	0x4001b0b0
-				MX7D_PAD_LCD_DATA10__LCD_DATA10	0x4001b0b0
-				MX7D_PAD_LCD_DATA11__LCD_DATA11	0x4001b0b0
-				MX7D_PAD_LCD_DATA12__LCD_DATA12	0x4001b0b0
-				MX7D_PAD_LCD_DATA13__LCD_DATA13	0x4001b0b0
-				MX7D_PAD_LCD_DATA14__LCD_DATA14	0x4001b0b0
-				MX7D_PAD_LCD_DATA15__LCD_DATA15	0x4001b0b0
-				MX7D_PAD_LCD_DATA16__LCD_DATA16	0x4001b0b0
-				MX7D_PAD_LCD_DATA17__LCD_DATA17	0x4001b0b0
-				MX7D_PAD_LCD_DATA18__LCD_DATA18	0x4001b0b0
-				MX7D_PAD_LCD_DATA19__LCD_DATA19	0x4001b0b0
-				MX7D_PAD_LCD_DATA20__LCD_DATA20	0x4001b0b0
-				MX7D_PAD_LCD_DATA21__LCD_DATA21	0x4001b0b0
-				MX7D_PAD_LCD_DATA22__LCD_DATA22	0x4001b0b0
-				MX7D_PAD_LCD_DATA23__LCD_DATA23	0x4001b0b0
+				MX7D_PAD_LCD_DATA00__LCD_DATA0	0x79
+				MX7D_PAD_LCD_DATA01__LCD_DATA1	0x79
+				MX7D_PAD_LCD_DATA02__LCD_DATA2	0x79
+				MX7D_PAD_LCD_DATA03__LCD_DATA3	0x79
+				MX7D_PAD_LCD_DATA04__LCD_DATA4	0x79
+				MX7D_PAD_LCD_DATA05__LCD_DATA5	0x79
+				MX7D_PAD_LCD_DATA06__LCD_DATA6	0x79
+				MX7D_PAD_LCD_DATA07__LCD_DATA7	0x79
+				MX7D_PAD_LCD_DATA08__LCD_DATA8	0x79
+				MX7D_PAD_LCD_DATA09__LCD_DATA9	0x79
+				MX7D_PAD_LCD_DATA10__LCD_DATA10	0x79
+				MX7D_PAD_LCD_DATA11__LCD_DATA11	0x79
+				MX7D_PAD_LCD_DATA12__LCD_DATA12	0x79
+				MX7D_PAD_LCD_DATA13__LCD_DATA13	0x79
+				MX7D_PAD_LCD_DATA14__LCD_DATA14	0x79
+				MX7D_PAD_LCD_DATA15__LCD_DATA15	0x79
+				MX7D_PAD_LCD_DATA16__LCD_DATA16	0x79
+				MX7D_PAD_LCD_DATA17__LCD_DATA17	0x79
+				MX7D_PAD_LCD_DATA18__LCD_DATA18	0x79
+				MX7D_PAD_LCD_DATA19__LCD_DATA19	0x79
+				MX7D_PAD_LCD_DATA20__LCD_DATA20	0x79
+				MX7D_PAD_LCD_DATA21__LCD_DATA21	0x79
+				MX7D_PAD_LCD_DATA22__LCD_DATA22	0x79
+				MX7D_PAD_LCD_DATA23__LCD_DATA23	0x79
 			>;
 		};
 
 		pinctrl_lcdif_ctrl: lcdifctrlgrp {
 			fsl,pins = <
-				MX7D_PAD_LCD_CLK__LCD_CLK	0x4001b0b0
-				MX7D_PAD_LCD_ENABLE__LCD_ENABLE	0x4001b0b0
-				MX7D_PAD_LCD_VSYNC__LCD_VSYNC	0x4001b0b0
-				MX7D_PAD_LCD_HSYNC__LCD_HSYNC	0x4001b0b0
+				MX7D_PAD_LCD_CLK__LCD_CLK	0x79
+				MX7D_PAD_LCD_ENABLE__LCD_ENABLE	0x79
+				MX7D_PAD_LCD_VSYNC__LCD_VSYNC	0x79
+				MX7D_PAD_LCD_HSYNC__LCD_HSYNC	0x79
 			>;
 		};
 	};
-- 
1.7.5.4

